common/openapi.yml

```diff
@@ -202,3 +355,28 @@ components:
         - CANCELED
         - CANCELING
       type: string
+    RunSetState:
+      title: RunSetState
+      example: RUNNING
+      enum:
+        - SET_UNKNOWN_STATE
+        - SET_RUNNING
+        - SET_COMPLETE
+        - SET_ERROR
```

Comment:
Should this list be same as [WES responses as we had in RunState](https://github.com/DataBiosphere/cbas/blob/dc134dfc4793a3840903f74cd39cee39fad497ab/common/openapi.yml#L193-L203)?

Reply:
Happy to if you feel strongly... I thought it would help to make them an obviously different set, to help distinguish between the status types?

Reply:
Would it make sense to add `QUEUED`, `CANCELING` `CANCELLED` statuses? This way when a request is accepted it is put to Queued state and then when picked up to Running state? Also adding the other 2 statuses will support aborting the run set?

Reply:
I'm not sure... I was going to leave those out until we implemented the functionality behind them